dvc/stage/__init__.py

```diff
@@ -568,5 +554,8 @@ def _status_stage(self, ret):
 
     def changed_stage(self, warn=False):
```

Comment:
But this should not only check for cmd but also if some deps/outs were added or deleted, no?

Reply:
Yes, the naming is a bit confusing.

Reply:
The naming is this way because `PipelineStage` need to check for `cmd` and this `Stage` checks for `md5`.